local-cli/server/server.js

```diff
@@ -13,7 +13,11 @@ const formatBanner = require('./formatBanner');
 const path = require('path');
 const runServer = require('./runServer');
 const findSymlinksPaths = require('./findSymlinksPaths');
-const NODE_MODULES = path.resolve(__dirname, '..', '..', '..');
+
+const NODE_MODULES =
+  /node_modules/.test(__dirname) ?
+    path.resolve(__dirname, '..', '..', '..') :
+    path.resolve('node_modules');
```

Comment:
From the [docs](https://nodejs.org/docs/latest/api/path.html#path_path_resolve_paths):

The path.resolve() method resolves a sequence of paths or path segments into an absolute path.

What's the use case for using it with a single argument? Is this line correct?